README.md

```diff
@@ -44,27 +60,27 @@ Next, check **[First Time Setup](#first-time-setup)** to continue setting up you
 
 #### Connect your store with your Omise account
 
-![Omise-WooCommerce plugin - setting page](https://user-images.githubusercontent.com/2154669/62671292-7b991480-b9c0-11e9-8627-0bc85c078365.png)
+![Omise-WooCommerce plugin - setting page](https://user-images.githubusercontent.com/2154669/68251435-b3face00-0055-11ea-9932-d2129bc3f1e7.png)
 
 Once the installation is done, the next thing that you are going to do is to connect your store with your Omise account and enable payment methods so your customers can make a purchase with an online payment.
 
 1. Log in to WordPress admin page.
 
 2. From the sidebar, go to `Omise > Settings`.
 
-3. At the Omise Settings page, you are going to set your `Public key` and `Secret key` (these keys can be found at Omise Dashboard).
+3. At the Omise Settings page, you are going to set your `Public key` and `Secret key` (these keys can be found at [Omise Dashboard](https://dashboard.omise.co/test/keys)).
 
 4. Make sure that the option `Test mode` is checked and set your Omise keys at `Public key for test` and `Secret key for test` fields if you would like to test Omise service integration.
 
 5. Click **'Save Settings'**.
 
 #### Enable payment methods
 
-> Note that Alipay, Bill Payment: Tesco, Installment, and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
+> Note that Alipay, Bill Payment: Tesco, Installment, Internet Banking, and TrueMoney Wallet payment methods are only available for merchants with a Thai-registered Omise account.
 
-There are 5 payment methods that you can enable on your store, which are **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, and **Internet Banking**.
+There are 6 payment methods that you can enable on your store, which are **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, **Internet Banking**, and **TrueMoney Wallet**.
```

Comment:
Just for the sake of laziness, I'd avoid writing the number "6" explicitly here as it's inevitable that (1) it's going to change (2) we're going to forget to update it.

Reply:
Removing the number sounds good. But cannot think of any better sentence here at the moment... 
@danfowler any suggestion? 🆘

Reply:
The payment methods available for you to enable on your store are: blah, blah, blah, and blah.